Quarg Consider their Outfits and Ships Atrocities (#7737)
endless-sky/endless-sky: data/quarg/quarg.txt

		"You have only a short time to learn to live together in peace."
		"You must live in peace... or leave the galaxy."

conversation "quarg imprisonment"

> 	`Your ship loses all power immediately upon touching down on the landing pad. As you walk around to try and find the problem, you pass by your hatch, already opened, and with dozens of Quarg coming in. They are all clad in elegant full-body armor suits.`

Reviewer: ```suggestion
	`Your ship loses all power immediately upon touching down on the landing pad. As you walk around to try and find the problem, you pass by your hatch. You find that it is already opened with dozens of Quarg coming in, all clad in elegant full-body armor suits.`
```

> 	`Your ship loses all power immediately upon touching down on the landing pad. As you walk around to try and find the problem, you pass by your hatch, already opened, and with dozens of Quarg coming in. They are all clad in elegant, clean full-body armor suits.`

Reviewer 2: ```suggestion
	`Your ship loses all power immediately upon touching down on the landing pad. As you walk around to try and find the problem, you pass by your hatch, already opened, and with dozens of Quarg coming in. They are all clad in elegant, clean, full-body armor suits.`
```

	`	"<first> <last>, you are found to be in illegal possession of Quarg equipment. Our devices are precious, and ours alone. Though it is regrettable, we must seize you."`

> 	`	They don't allow you to get as much as a word out before one of the guards has locked your arms behind your back, completely halting your attempts to fight back with surprisingly little effort. You're brought before something akin to a Quarg jury, where you are lectured on for several hours on how dangerous it is for Quarg apparatus to be stolen and used so lightly by others. At the end of it all, they sentece you to life imprisonment.`

Reviewer 2: ```suggestion
	`	They don't allow you to get as much as a word out before one of the guards has locked your arms behind your back, completely halting your attempts to fight back with surprisingly little effort. You are brought before something akin to a Quarg jury, where they lecture you for several hours on how dangerous it is for Quarg apparatus to be stolen and used so lightly by others. At the end of it all, they sentence you to life imprisonment.`
```

I think "you are" sounds more formal and imposing to start off the sentence. The second change is just to avoid duplication of the "you are", and is pretty much optional. I think it sounds better, but it's not a required change. The last change is the spelling of "sentence" at the end.

		"You have only a short time to learn to live together in peace."
		"You must live in peace... or leave the galaxy."

conversation "quarg imprisonment"
	`Your ship loses all power immediately upon touching down on the landing pad. As you walk around to try and find the problem, you pass by your hatch. You find that it is already opened with dozens of Quarg coming in, all clad in elegant full-body armor suits.`

> 	`	"<first> <last>, you are found to be in illegal possession of Quarg equipment. Our devices are precious, and ours alone. Though it is regrettable, we must seize you."`

Reviewer 3: ```suggestion
	`	"<first> <last>, you are found to be in illegal possession of Quarg equipment. Our devices are precious and ours alone. Though it is regrettable, we must seize you."`
```

	`	They don't allow you to get as much as a word out before one of the guards has locked your arms behind your back, completely halting your attempts to fight back with surprisingly little effort. You are brought before something akin to a Quarg jury, where they lecture you for several hours on how dangerous it is for Quarg apparatus to be stolen and used so lightly by others. At the end of it all, they sentence you to life imprisonment.`

> 	`	You spend the rest of your days in the same cell, which although large and comfortable, quickly proves plain and dull. You're kept in very good health, and are well fed, but the Quarg do not allow you to leave the cell, and seldom entertain your attempts at striking conversation. You get few visitors, and can only wonder what life would've been like had you the chance to live out all these years in freedom, the captain of a starship.`

Reviewer 3: ```suggestion
	`	You spend the rest of your days in the same cell, which although large and comfortable, quickly proves plain and dull. You're kept in very good health and are well fed, but the Quarg do not allow you to leave the cell and seldom entertain your attempts at striking conversation. You get few visitors and can only wonder what life would've been like had you the chance to live out all these years in freedom, the captain of a starship.`
```
